src/generalizedlinearmixedmodel.jl

```diff
@@ -355,6 +355,8 @@ function GeneralizedLinearMixedModel(
         LMM = LinearMixedModel(
             LMM.formula,
             LMM.allterms,
+            LMM.reterms,
```

Comment:
I'm wondering if it wouldn't be easier to just modify the already constructed LMM. There's not much overhead here since we're side-stepping the constructor and not doing any extra allocations, but we could replace the weights and response fields in the existing struct (I just checked and `LinearMixedModel` is *not* `immutable`.)